### PR TITLE
Mel 789/final documentation updates

### PR DESCRIPTION
**Pull request by kathrynnapier**
Naming changes throughout the workflows and project documentation to include the full name of the project, references to pilot project and pilot project dashboard partners. Also included changes to the github README to include details of other required repositories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,5 +28,14 @@ Oapen Metadata, Onix, UCL Discovery and an Onix Workflow for combining all of th
...
 | <img src="docs/logos/ucl.svg" alt="UCL Discovery" width="150" /> | UCL Discovery is UCL's open access repository, showcasing and providing access to the full texts of UCL research publications.The metadata for all eprints is obtained from their publicly available CSV file (https://discovery.ucl.ac.uk/cgi/search/advanced).  |
 
+
 ## Documentation
-For detailed documentation about the OAeBU Workflows see the Read the Docs website [https://oaebu-workflows.readthedocs.io](https://oaebu-workflows.readthedocs.io)
+For detailed documentation about the Book Usage Data Workflows see the Read the Docs website  [https://oaebu-workflows.readthedocs.io](https://oaebu-workflows.readthedocs.io)
+
+## Requirements
```
Comment (aroelo) [inline README.md:35]: I'm not sure in what sense these are 'requirements', one way of looking at it is that these repositories are all used to create the final datasets for the Book Usage Data Workflows project, but the bottom 2 are not technically a requirement to install/use this specific repository.

I would maybe rename it to 'related work' or 'associated repositories', or something along those lines. Or maybe making it a bit longer to clarify, 'Other requirements to create the Book Usage Dataset', not sure.

File: README.md [hunk 2]
```diff
@@ -28,5 +28,14 @@ Oapen Metadata, Onix, UCL Discovery and an Onix Workflow for combining all of th
...
+The Observatory Platform, an environment for fetching, processing and analysing data, see the Repository  [https://github.com/The-Academic-Observatory/observatory-platform](https://github.com/The-Academic-Observatory/observatory-platform)
+
+The Academic Observatory Workflows, which provides Apache Airflow workflows for fetching, processing and analysing data about academic institutions, see the Repository  [https://github.com/The-Academic-Observatory/academic-observatory-workflows](https://github.com/The-Academic-Observatory/academic-observatory-workflows)
+
+The Onix Parser, a command line tool that transforms ONIX files into a format suitable for loading into BigQuery, see see the Repository  [https://github.com/The-Academic-Observatory/onix-parser](https://github.com/The-Academic-Observatory/onix-parser)
```
Comment (aroelo) [inline README.md:40]:
```suggestion
The Onix Parser, a command line tool that transforms ONIX files into a format suitable for loading into BigQuery, see the Repository  [https://github.com/The-Academic-Observatory/onix-parser](https://github.com/The-Academic-Observatory/onix-parser)
```

File: docs/README.md [hunk 3]
```diff
@@ -10,3 +10,13 @@
 6) Install documentation dependencies: `pip install -r requirements.txt`
 7) Build documentation: `make html`
 8) To view the documentation, open `_build\html\index.html`.
+
+## MacOS
+1) Install Python 3.8 
+3) In the terminal, navigate to the `oaebu-workflows\docs` folder.
```
Comment (aroelo) [inline docs/README.md:16]:
```suggestion
3) In the terminal, navigate to the `oaebu-workflows/docs` folder. 
```

File: docs/README.md [hunk 4]
```diff
@@ -10,3 +10,13 @@
 6) Install documentation dependencies: `pip install -r requirements.txt`
 7) Build documentation: `make html`
 8) To view the documentation, open `_build\html\index.html`.
+
+## MacOS
+1) Install Python 3.8 
+3) In the terminal, navigate to the `oaebu-workflows\docs` folder. 
+3) Create virtual env, if using conda: `conda create -n <nameofvenv> python=3.8 anaconda`
+4) Activate virtual env: `conda activate <nameofvenv>`
+5) Add docs directory directory to Python path: `PYTHONPATH="${PYTHONPATH}:/Users/278559c/Desktop/oaebu-workflows/docs"` followed by `export PYTHONPATH`
```
Comment (aroelo) [inline docs/README.md:19]:
```suggestion
5) Add docs directory directory to Python path: `PYTHONPATH="${PYTHONPATH}:/Users/MyUser/Documents/oaebu-workflows/docs"` followed by `export PYTHONPATH`. Make sure to replace `MyUser` with the name of your user.
```

File: docs/README.md [hunk 5]
```diff
@@ -10,3 +10,13 @@
...
+6) Install documentation dependencies: `pip install -r requirements.txt`
+7) Build documentation: `make html`
+8) To view the documentation, open `_build\html\index.html`.
```
Comment (aroelo) [inline docs/README.md:22]:
```suggestion
8) To view the documentation, open `_build/html/index.html`.
```

**Review by aroelo — CHANGES_REQUESTED** [review]
Thanks @kathrynnapier, just some very minor suggested changes

File: README.md (hunk 1 above)
Comment (kathrynnapier) [reply README.md]: Thanks @aroelo I was keen for feedback on the wording of this. I've updated to: 'Other requirements to create the Book Usage Dataset',

File: docs/README.md (hunk 3 above)
Comment (kathrynnapier) [reply docs/README.md]: Thanks @aroelo ! Done.

**Review by aroelo — APPROVED** [review]
Thanks @kathrynnapier!

**Pull request merged**